src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/TentativeInstanceMethodNode.cs

```diff
@@ -35,11 +35,11 @@ public TentativeInstanceMethodNode(IMethodBodyNode methodNode)
 
         public override bool HasConditionalStaticDependencies => true;
 
-        protected override ISymbolNode GetTarget(NodeFactory factory)
+        public override IMethodNode GetTarget(NodeFactory factory)
```

Comment:
This visibility change seems harmless to me, but maybe @jkotas you'd like to cast an eye on it?

Reply:
It looks good to me. @MichalStrehovsky Any concerns?

Reply:
Nope, looks good. All AssemblyStubNode descendants need some special treatment because on WASM we're not able to generate byte buffers with extra code in it (which is what we do elsewhere).